src/main/java/com/epam/pages/popup/AcademicClassPopup.java

```diff
@@ -1,6 +1,48 @@
 package com.epam.pages.popup;
 
-import com.epam.pages.common.CommonPopup;
+import com.epam.helpers.ErrorMessagesProvider;
+import com.epam.helpers.SharedTestData;
+import com.epam.helpers.UserDataProvider;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
-public class AcademicClassPopup extends CommonPopup {
+import java.util.List;
+
+public class AcademicClassPopup extends CreatePopup {
```

Comment:
Please extends this class from CommonPopup, becauese you needn't somthing from CreatePopup.

Reply:
done